lambdas/db-migration/src/migrations/20210504103649_remove_report_to_ems.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
```

Comment:
Should we consider including the table name/names afffected in the migration file name?        I'm not envisioning a *ton* of migrations in this directory, however it might make tracking down changes easier in the future.     Not blocking, but for discussion.

Reply:
Yes, I think that's a good idea. Would be nice to be able to tell which table is being changed at a glance.

Reply:
👍🏻  that's a great name change, thanks for updating: 83fea152336e1c772418b000edf15fcb9be1f142